datalad_release_action/versions.py

```diff
@@ -68,8 +68,10 @@ def bump_version(v: str, bump: Bump) -> str:
     '1.2.4'
     >>> bump_version("1.2.3.4", Bump.PATCH)
     '1.2.4.0'
+    >>> bump_version("v1.2.3", Bump.PATCH)
+    '1.2.4'
     """
-    parts = [int(p) for p in v.split(".")]
+    parts = [int(p) for p in v.lstrip("v").split(".")]
```

Comment:
hm, to not postpone proper solution, shouldn't `tag-prefix` be made into a configuration file option? 
and then passed into this function to be stripped since if we decide to use this action for some other tricky cases (dandischema?) where we do have other prefixes, this ad-hoc workaround would no longer work.

Reply:
I feel that's out of scope for this PR.  File a separate issue if you want that.

Note that dandischema's `schema-*` tags won't cause the action to crash, as any tags that aren't of the form N.N.N or vN.N.N are currently discarded.

Reply:
ok, let's just consider that the only supported `tax-prefix` is `"v"` for now ;)

will file an issue if we run into the use-case